src/test/scala/nl/knaw/dans/api/sword2/ResolveFetchItemsSpec.scala

```diff
@@ -109,9 +109,37 @@ class ResolveFetchItemsSpec extends Sword2Fixture with BagStoreFixture {
   }
 
   it should "result in a Failure when the bag-store base-dir doesn't exist"  in {
+    implicit val bagStoreBase = BagStoreBase("non/existent/dir", baseUrl)
     copyToTargetBagDir(SIMPLE_SEQUENCE_A)
-    implicit val baseDir = new File("non/existent/dir")
     DepositHandler.checkBagStoreBaseDir() shouldBe a[Failure[_]]
   }
+
+  it should "result in a Success when both bag-store base-dir and base-uri are not given, and there are no fetch.txt references to the bag-store"  in {
+    implicit val bagStoreBase = BagStoreBase("", "")
+    copyToTargetBagDir(SIMPLE_SEQUENCE_A)
+    DepositHandler.setBagStoreAwareness shouldBe a[Success[_]]
+    DepositHandler.checkBagStoreBaseDir() shouldBe a[Success[_]]
+    DepositHandler.checkFetchItemUrls(targetBagDir, urlPattern) shouldBe a[Success[_]]
+    DepositHandler.checkBagVirtualValidity(targetBagDir) shouldBe a[Success[_]]
+  }
+
+  it should "result in a Failure when both bag-store base-dir and base-uri are not given, and there are fetch.txt references to the bag-store"  in {
+    implicit val bagStoreBase = BagStoreBase("", "")
+    copyToTargetBagDir(SIMPLE_SEQUENCE_B)
+    DepositHandler.setBagStoreAwareness shouldBe a[Success[_]]
+    DepositHandler.checkBagStoreBaseDir() shouldBe a[Success[_]]
+    DepositHandler.checkFetchItemUrls(targetBagDir, urlPattern) shouldBe a[Success[_]]
+    DepositHandler.checkBagVirtualValidity(targetBagDir) shouldBe a[Failure[_]]
```

Comment:
We experimented a bit with this after we found that this takes quite a long time for the test to succeed (because of the downloads when the bag-store does not exists, see previous comment):

```scala
val validity = DepositHandler.checkBagVirtualValidity(targetBagDir)

validity shouldBe a[Failure[_]]
    
type expectedException = IllegalArgumentException

val theException = the [expectedException] thrownBy validity.get
theException.getMessage shouldBe "blabla"

a [expectedException] should be thrownBy validity.get
```